src/components/Filters/Filters.tsx

```diff
@@ -231,6 +231,7 @@ class FiltersInner extends Component<CombinedProps, State> {
       <div ref={this.moreFiltersButtonContainer}>
         <Button
           onClick={this.toggleFilters}
+          id="SheetToggleButton"
```

Comment:
We need to avoid fixed IDs. Using a fixed ID means we can end up with accessibility warnings about "you have duplicate ids on the page" if you ever render two Filters components on one page.

Can you remove this id and target the element in a different way?